src/python/zquantum/core/utils.py

```diff
@@ -440,8 +443,26 @@ def create_object(specs: Dict, **kwargs):
     module = importlib.import_module(module_name)
     creator_name = specs.pop("function_name")
     creator = getattr(module, creator_name)
-    created_object = creator(**specs, **kwargs)
-    return created_object
+
+    for key in specs.keys():
+        if key in kwargs.keys():
+            raise ValueError("Cannot have same parameter assigned to multiple values")
+
+    if isinstance(creator, FunctionType):
+        if kwargs != {} or specs != {}:
+            import inspect
```

Comment:
Is there a reason why the import is here and not on the top of the file?

Reply:
Nope! Fixed here: 96d45a1